### PR TITLE
Added assignment 2b and assignment 4

### PR DESCRIPTION
**Pull request by paumurl**

### CODE REVIEW

**Comment by dgarijo** [comment]
@ocorcho

**Comment by ocorcho** [comment]
Mine are ok. 
@dgarijo check whether there is anything else to check in assignment 4, and accept when ready

File: Assignment4/paumurl-e22a302/Task06.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# **TASK 6.5: Add UPM as the university where John Smith works**
+
+# In[8]:
+
+
+# TO DO
+# Visualize the results
+from rdflib.namespace import FOAF
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, FOAF.member, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/paumurl-e22a302/Task06.py:102]: the domain of foaf:member is a Group, so this is not correct. (the property is intended to be ns.UPM foaf:member ns:JohnSmith). Well done for searching for alternative properties, but you have to make sure they match your domain/range. If you cannot find one, create your own.

File: Assignment4/paumurl-e22a302/Task08.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        g1.add((s1,p1,o1))
+
+
+# In[6]:
+
+
+#Comprobamos que se ha completado correctamente
+for s,p,o in g1.triples((None, RDF.type, ns.Person)):
+    for s1,p1,o1 in g1.triples((s,None,None)):
+      print(s1,p1,o1)
```
Comment (dgarijo) [inline Assignment4/paumurl-e22a302/Task08.py:61]: You completed all fields, not the fields you were requested.

**Review by dgarijo — CHANGES_REQUESTED** [review]
Well done in general. Small changes requested

File: Assignment4/paumurl-e22a302/Task06.py (hunk 1 above)
Comment (dgarijo) [inline Assignment4/paumurl-e22a302/Task06.py:102]: Still not correct

**Review by dgarijo — CHANGES_REQUESTED** [review]
Still unaddressed comments

**Review by dgarijo — APPROVED** [review]

**Pull request merged**